.github/ISSUE_TEMPLATE/found-dead-links.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Found Dead Links
+about: Dead link
+title: 'Dead link(s) found in markdown files'
+labels: 'Documentation'
+assignees: ''
+
+---
+
+To find the dead links, run the following commands in the top-level directory:
+- npm install -g markdown-link-check
```

Comment:
most java and c# devs won't have npm installed

Can you redirect the ci-cd to a file and use that to create the issue?

Reply:
I put lines 10 and 11 in to help the developer find the dead links. They can find them by looking at the logs under GitHub Actions, so these lines  are not necessary. I'll work on creating the issue from a file, and see if the logs with the dead links can be written to the file.